packages/web-components/fast-ssr/src/fast-style/index.ts

```diff
@@ -0,0 +1,78 @@
+import { DOM, StyleTarget } from "@microsoft/fast-element";
```

Comment:
Let's re-work this so there's no dependency on fast-element at all. I think it's ok to just duplicate the `supportsAdoptedStyleSheets` code in order to keep this dependency free. It's a pretty small little snippet.